examples/kafka-hub/integration_test_suite/main.bal

```diff
@@ -94,45 +74,22 @@ isolated function registerTopic(websubhub:PublisherClient publisherClientEp) ret
     return publisherClientEp->registerTopic(TOPIC);
 }
 
-isolated function subscribe(websub:Listener websubListener, websub:SubscriberService subscriberService) returns error? {
-    websub:SubscriberServiceConfiguration config = {
-        target: [HUB, TOPIC],
-        callback: CALLBACK_URL,
-        secret: SECRET,
-        unsubscribeOnShutdown: true
-    };
-    check websubListener.attachWithConfig(subscriberService, config);
-    return websubListener.'start();
-}
-
-isolated function publishContent(websubhub:PublisherClient publisherClientEp) returns int {
-    int sentCount = 0;
+isolated function publishContent(websubhub:PublisherClient publisherClientEp) returns error? {
     foreach json message in messages {
-        sentCount += 1;
-        var publishResponse = publisherClientEp->publishUpdate(TOPIC, message);
-        if publishResponse is websubhub:UpdateMessageError {
-            log:printWarn("Error occurred while publishing content");
-        }
+        _ = check publisherClientEp->publishUpdate(TOPIC, message);
     }
-    return sentCount;
-}
-
-isolated function unsubscribe(websub:Listener websubListener) returns error? {
-    return websubListener.gracefulStop();
 }
 
 isolated function deregisterTopic(websubhub:PublisherClient publisherClientEp) returns websubhub:TopicDeregistrationSuccess|error {
     return publisherClientEp->deregisterTopic(TOPIC);
 }
 
-isolated function writeResultsToCsv(any[][] results, string output_path) returns error? {
-    string[][] final_results = [];
-    foreach any[] resultLine in results {
-        string[] constructedResultLine = [];
-        foreach var result in resultLine {
-            constructedResultLine.push(result.toString());
-        }
-        final_results.push(constructedResultLine);
+function writeResultsToCsv(string output_path, any[] results) returns error? {
```

Comment:
```suggestion
function writeResultsToCsv(string outputPath, any[] results) returns error? {
```

Reply:
Please correct other places as well. We usually do not use `_`

Reply:
Fixed with [81af5e0c7eaca4b82b848a1da3b4f66df58882c8](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/724/commits/81af5e0c7eaca4b82b848a1da3b4f66df58882c8)